cmd/toolkit/smart_contract.go

```diff
@@ -6,13 +6,14 @@ import (
 	"fmt"
 	"math/big"
 
```

Comment:
This file implements the contract calls of our toolkit CLI. The changes here are just to use the Tables Ethereum Client